tuxemon/event/conditions/npc_facing_tile.py

```diff
@@ -56,6 +58,17 @@ def test(self, session: Session, condition: MapCondition) -> bool:
         # get all the coordinates around the npc
         client = session.client
         npc_tiles = get_coords(npc.tile_pos, client.map_size)
+
+        # check if the NPC is facing a specific set of tiles
+        world = session.client.get_state_by_name(WorldState)
+        if len(condition.parameters) > 1:
+            value = condition.parameters[1]
+            if value == "surfable":
```

Comment:
ok for now, but in general, "surfable" tiles is a feature for the spyder campaign, and we should have a way to get all tiles which have a specific key/value, rather than hardcoding it